Xamarin.Forms.Platform.iOS/Renderers/SwipeViewRenderer.cs

```diff
@@ -13,6 +13,7 @@ namespace Xamarin.Forms.Platform.iOS
 {
 	public class SwipeViewRenderer : ViewRenderer<SwipeView, UIView>
 	{
+		const float MinimumOpenSwipeThresholdPercentage = 0.15f; // 15%
```

Comment:
Scrolling quickly, we can slightly open the SwipeView. For that reason, we do not block the scroll unless the SwipeView is opened 15%.